README.md

```diff
@@ -47,184 +45,208 @@ http://ryanstutorials.net/linuxtutorial/
  On Redhat or CENTOS systems, these can all be installed with the following command:
  ```sh
    sudo yum update
-    sudo yum install make automake gcc gcc-c++ kernel-devel python-devel openssl openssl-devel libevent-devel git
+   sudo yum install make automake gcc gcc-c++ kernel-devel python-devel openssl openssl-devel libevent-devel git
  ```
 
+ ###2. Clone VOLTTRON code
 
-**2. Install Erlang version 21 packages.**
-
-
- For RabbitMQ based VOLTTRON, some of the RabbitMQ specific software packages have to be installed.
-
-  **On Debian and CentOS 6/7**
-
-  If you are running an Debian or CentOS system, you can install the RabbitMQ dependencies by running the rabbit 
-  dependencies script, passing in the os name and approriate distribution as a parameter. The following are  
-  supported
-
-    debian bionic (for Ubuntu 18.04)
-    debian xenial (for Ubuntu 16.04)
-    debian xenial (for Linux Mint 18.04)
-    debian stretch (for Debian Stretch)
-    centos 7 (for CentOS 7)
-    centos 6 (for CentOS 6)
-  
-  Example command
-  ``` 
-  ~/volttron/scripts/rabbit_dependencies.sh debian xenial
-  ```
-  
-  **Alternatively**
-  
-  You can also download and install Erlang from [Erlang Solutions](https://www.erlang-solutions.com/resources/download.html).
-  Please include OTP/components - ssl, public_key, asn1, and crypto.
-  Also lock version of Erlang using the [yum-plugin-versionlock](https://access.redhat.com/solutions/98873)
-
-**3. Configure hostname**
-
-Make sure that your hostname is correctly configured in /etc/hosts.
-See (https://stackoverflow.com/questions/24797947/os-x-and-rabbitmq-error-epmd-error-for-host-xxx-address-cannot-connect-to-ho). If you are testing with VMs make please make sure to provide unique host names for each of the VM you are using. 
-
-Hostname should be resolvable to a valid ip when running on bridged mode. RabbitMQ checks for this during
-initial boot. Without this (for example, when running on a VM in NAT mode)
-RabbitMQ  start would fail with the error "unable to connect to empd (
-port 4369) on <hostname>." Note: RabbitMQ startup error would show up in syslog (/var/log/messages) file
-and not in RabbitMQ logs (/var/log/rabbitmq/rabbitmq@hostname.log)
-
-
-**4. Download VOLTTRON code from experimental branch**
+From version 6.0 VOLTTRON supports two message bus - ZMQ and RabbitMQ. Clone the appropriate
+branch of volttron code. The develop branch currently only supports ZMQ. The 6.0alpha branch
```

Comment:
After 6.0 release, master and develop branches will have ZMQ and RMQ. So this line does not make sense here.

Reply:
Fixed. Deleted contents after first line